core/lowering/passes/alias_op_mapping.cpp

```diff
@@ -0,0 +1,30 @@
+#include <torch/csrc/jit/passes/subgraph_rewrite.h>
+
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace lowering {
+namespace passes {
+
+void AliasOpMapping(std::shared_ptr<torch::jit::Graph>& graph) {

```

Comment:
The name `AliasOpMapping` seems very generic. Can you rename it to be more specific like maybe `TrueDivideMapping` ? Please add a test case for this pass in a new directory `//tests/core/lowering`. 
Rest of the changes LGTM

Reply:
Hi Dheeraj, I have moved true_divide from lowering phase to converters, and add support to other multiple converters. All those converters are provided with test case.